lib/twitter_cldr/js/mustache/data/calendars/timespan.coffee

```diff
@@ -0,0 +1,7 @@
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# TwitterCldr.TimespanFormatter.patterns = `{{{patterns}}}`
+
+  TimespanFormatter :
```

Comment:
Since now data templates are much simpler in terms of structure, maybe we could get rid of these CoffeeScript templates and instead generate and render data for each class (either individually or all together) using a simple call to the JSON library? We can get the data from each data renderer as a Ruby hash, merge them all together (still in Ruby), and then render them all at once using `JSON.dump` or just one simple template that optionally defines `TwitterCldr` and then sets `TwitterCldr.data` (if we choose to go in that direction, of course; see my comments above). That way these data renderers don't even need to be renderers anymore – they'll be just returning the data.